utils/python/CIME/XML/env_batch.py

```diff
@@ -4,7 +4,7 @@
 from standard_module_setup import *
 
 from env_base import EnvBase
-
+from CIME.utils import convert_to_string
```

Comment:
Good question. I had one regression test fail - and when Jim provided me this fix to get the tests to pass.
I think he needs to comment. It was not tied directly to my changes - only tied to getting the tests to pass.